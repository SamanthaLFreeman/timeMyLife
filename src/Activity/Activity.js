import React from 'react';

const Activity = ({activity}) => {
  return (
    <article className='activityCard'>
      <h3 className='activityText'>{activity.category}</h3>
      <p className='activityText'>{activity.mins} MIN {activity.secs} SECONDS</p>
      <p className='activityText'>{activity.name}</p>
    </article>
  )
}

export default Activity;