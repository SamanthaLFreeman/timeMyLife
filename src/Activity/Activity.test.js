import React from 'react';
import Activity from './Activity';
import { shallow } from 'enzyme';

describe('Activity', () => {
  let wrapper;
  const activityMock = {
    category: 'Study',
    name: 'Activity name',
    mins: '1',
    secs: '30'
  }
  beforeEach(() => {
    wrapper = shallow(<Activity 
    activity={activityMock} />)
  })

  it('should match the snapshot with the correct data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  })
})