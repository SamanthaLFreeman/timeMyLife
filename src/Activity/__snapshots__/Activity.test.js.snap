// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activity should match the snapshot with the correct data passed through 1`] = `
<article
  className="activityCard"
>
  <h3
    className="activityText"
  >
    Study
  </h3>
  <p
    className="activityText"
  >
    1
     MIN 
    30
     SECONDS
  </p>
  <p
    className="activityText"
  >
    Activity name
  </p>
</article>
`;
